releasenotes/notes/pauli_evolve_rz-f6d9fc0cabd8a183.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    :meth:`.Pauli.evolve` now correctly handles quantum circuits containing certain parametrized rotation gates,
+    when the angle is a multiple of pi/2, e.g. :class:`.RZGate(np.pi/2)` or :class:`.RZZGate(np.pi/2)`.
```

Links are for the classes (i.e. `:class:RZGate`) not for instances like `RZGate(0.1)`. Since you're just naming example instances here we can just omit the link
```suggestion
    when the angle is a multiple of pi/2, e.g. `RZGate(np.pi/2)` or `RZZGate(np.pi/2)`.
```

---

done in [d85c38d](https://github.com/Qiskit/qiskit/pull/15289/commits/d85c38d9ca2f839c9e30034412bdbb4139b5e200)